server/storages/factory.go

```diff
@@ -145,6 +151,10 @@ func (f *FactoryImpl) Create(name string, destination DestinationConfig) (Storag
 		for _, field := range destination.DataLayout.PrimaryKeyFields {
 			pkFields[field] = true
 		}
+
+		if destination.DataLayout.MaxColumns > 0 {
```

Comment:
There must be a default value = 100, and the destination can override it.

Reply:
Message was added